lib/indent-codecrumbs-view.js

```diff
@@ -16,7 +16,7 @@ export default class IndentCodeCrumbsView {
         {
           this.props.crumbs.map(_ =>
             <li on={{click: () => this.props.crumbsClick(_)}} className={`indent-codecrumbs__el ${this.props.color ? _.scopeClasses : ""}`}>
-              {_.title}
+              <span>{_.title}</span>
```

Comment:
We are heading to use best practices for frontend developing, like BEM naming in CSS class names (for details see: https://en.bem.info/methodology/quick-start/#contents)

```suggestion
              <span className='indent-codecrumbs__crumb-text'>{_.title}</span>
```